data/hulls/wing_data.csv

```diff
@@ -2,19 +2,19 @@ id,variant,tags,tier,rarity,fleet pts,op cost,formation,range,attackRunRange,att
 diableavionics_arbitrator_wing,diableavionics_arbitrator_standard,"interceptor2, interceptor, high, diable_ship_bp",0,,5,6,CLAW,6000,250,-200,3,INTERCEPTOR,Rapid Interception,10,6000,,,,,,,,,,,,,7
 diableavionics_arbitratorZ_wing,diableavionics_arbitratorZ_rocketeer,"bomber4, bomber, high, diable_ship_bp",1,,6,12,BOX,4000,2000,300,3,BOMBER,Rocketeer,15,11000,,,,,,,,,,,,,7
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-diableavionics_frost_wing,diableavionics_frost_Standard,"wanzer, fighter1, fighter, high, diable_wanzer_bp, rare_bp, rd_no_extra_craft",0,,6,8,CLAW,4000,500,,3,FIGHTER,Trooper,12.5,10000,,,,,,,,,,,,,1
-diableavionics_strife_wing,diableavionics_strife_Standard,"wanzer, fighter2, fighter, high, diable_wanzer_bp, rare_bp, rd_no_extra_craft",1,,7,12,BOX,4000,500,-100,2,FIGHTER,Gunner,15,11000,,,,,,,,,,,,,4
-diableavionics_hoar_wing,diableavionics_hoar_akimbo,"wanzer, fighter3, fighter, high, diable_wanzer_bp, rare_bp, rd_no_extra_craft",1,,7,10,BOX,1000,500,200,2,INTERCEPTOR,Bodyguard,15,11500,,,,,,,,,,,,,4
+diableavionics_frost_wing,diableavionics_frost_Standard,"wanzer, fighter1, fighter, high, diable_wanzer_bp, rare_bp, rd_no_extra_craft",0,,6,8,CLAW,4000,500,,3,FIGHTER,Trooper,10,10000,,,,,,,,,,,,,1
+diableavionics_strife_wing,diableavionics_strife_Standard,"wanzer, fighter2, fighter, high, diable_wanzer_bp, rare_bp, rd_no_extra_craft",1,,7,12,BOX,4000,500,-100,2,FIGHTER,Gunner,13,11000,,,,,,,,,,,,,4
+diableavionics_hoar_wing,diableavionics_hoar_akimbo,"wanzer, fighter3, fighter, high, diable_wanzer_bp, rare_bp, rd_no_extra_craft",1,,7,10,BOX,1000,500,200,2,INTERCEPTOR,Bodyguard,12,11500,,,,,,,,,,,,,4
```

Comment:
Frost refit time reduced to 10 (from 12.5)
Strife refit time reduced to 13 (from 15)
Hoar refit time reduced to 12 (from 15)